This patch addes space between the Snooze and Dismiss buttons for the clock alarms. Helps to hit the snooze button and not the dismiss button when still half asleep.

11-15-09:
Updated for 1.3.1
Index: /usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
+++ /usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
@@ -136,7 +136,7 @@ AppAssistant.prototype.createRingStage =
 	this.controller.createStageWithCallback({
 		lightweight: true,
 		name: this.kRingStageName, 
-		"height": 200, 
+		"height": 275, 
 		soundclass: soundClass,
 		sound: soundFile
 	}, f, 'popupalert');		
Index: /usr/palm/applications/com.palm.app.clock/app/views/ring/ring-scene.html
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/app/views/ring/ring-scene.html
+++ /usr/palm/applications/com.palm.app.clock/app/views/ring/ring-scene.html
@@ -1,4 +1,4 @@
-<div class="system-notification" style="height: 200px; position: fixed; bottom: 0">
+<div class="system-notification" style="height: 275px; position: fixed; bottom: 0">
 	<div class="notification-message" x-palm-popup-content="" id="message">
 		<div class="notification-icon"></div>
 		<div class="notification-text">
@@ -13,5 +13,6 @@
 		</div>
 	</div>
 	<div id="close_button" x-mojo-tap-highlight="immediate" class="palm-notification-button affirmative"><span x-mojo-loc="">Dismiss</span></div>
+	<div style="height: 75px;"></div>
 	<div id="snooze_button" x-mojo-tap-highlight="immediate" class="palm-notification-button alternate"><span x-mojo-loc="">Snooze</span></div>
 </div>                 
\ No newline at end of file
